openapi.yaml

```
@@ -372,15 +375,13 @@ tags:
       ```
       ### Example API call for GWA Intraday LTC:
```

Perhaps we should say something like:

```suggestion
      ### Example API call for GWA Intraday LTC:
      To request LTC index tickers with a timestamp before `17th March 2020`.
```
... or something similar, I often get confused whether the example is from or till query when it is simply `timestamp`.